yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -155,10 +155,10 @@ private[spark] class Client(
 
       // Get a new application from our RM
       val newApp = yarnClient.createApplication()
-      val newAppResponse = newApp.getNewApplicationResponse()
-      appId = newAppResponse.getApplicationId()
+      val newAppResponse = newApp.getNewApplicationResponse
```

Comment:
Yes, there are, but IDEA kept bugging me all the time about calling getters with `()` as if mutation/computation happened. I'll revert it.